receiver/mysqlreceiver/client.go

```diff
@@ -182,6 +184,49 @@ func newMySQLClient(conf *Config) client {
 	}
 }
 
+func (c *mySQLClient) getInnodbStatusStats() (map[string]int64, error) {
+	/*
+		TODO: The NewInnodbStatusParser should be able to be created with the mySQLClient.
+	*/
+	innodbParser, err := parser.NewInnodbStatusParser()
+	if err != nil {
+		err := fmt.Errorf("could not create parser for innodb stats, %s", err)
+		return nil, err
+	}
+	var (
+		typeVar string
+		name    string
+		status  string
+	)
+
+	query := "SHOW /*!50000 ENGINE*/ INNODB STATUS;"
+	row := c.client.QueryRow(query)
+	mysqlErr := row.Scan(&typeVar, &name, &status)
+
+	// TODO: Suggest better value if there's an error for the metric.
+	if mysqlErr != nil {
+		err := fmt.Errorf("error querying the mysql db for innodb status")
```

Comment:
We are losing original `mysqlErr` here.  Since you are using `fmt.Errorf` already, you should include the original error in the string.

Like `fmt.Errorf("error querying the mysql db for innodb status: %v", mysqlErr)`